### PR TITLE
#1999 added testing stage to work package forms

### PR DESCRIPTION
**Pull request by superhvarn**
## Changes

added testing stage to work package forms and made some backend changes to store it within the database.

## Screenshots
Screenshot 2024-02-01 at 9 08 27 PM

Screenshot 2024-02-01 at 10 20 57 PM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #1999)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -6164,9 +6164,9 @@ __metadata:
   linkType: hard
 
 "caniuse-lite@npm:^1.0.0, caniuse-lite@npm:^1.0.30000981, caniuse-lite@npm:^1.0.30001109, caniuse-lite@npm:^1.0.30001125, caniuse-lite@npm:^1.0.30001503":
-  version: 1.0.30001513
-  resolution: "caniuse-lite@npm:1.0.30001513"
-  checksum: 7d783f4f40584e7bdeda3f3abc23ffff21f14c37e1fadf1a6b3d2e299727fe9d5a61096101d9d4e4174f881d4ff4c62204fc41f3bc950f4da1b0edfdea025fa4
+  version: 1.0.30001582
+  resolution: "caniuse-lite@npm:1.0.30001582"
+  checksum: 2fc420cb6e6080a9808781ff81a2f0d37d63897c8c981d477001be18e55c8ec33422cf966e49efdca61d4a5335d16a4d6a09d5bd6da22a8396d0dcd350b9b9da
```
Comment (Aaryan1203) [inline yarn.lock:6169]: don't push the yarn lock file

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
left a small comment, but looks good

**Review by Aaryan1203 — CHANGES_REQUESTED** [review]
Also add a picture that shows the testing stage in the create work package view

**Review by RChandler234 — APPROVED** [review]
LGTM 🔥

**Pull request merged**